src/opening-period/page/CreateNewOpeningPeriodPage.test.tsx

```diff
@@ -0,0 +1,252 @@
+import React from 'react';
+import { act } from 'react-dom/test-utils';
+import { render, screen, fireEvent } from '@testing-library/react';
+import { DatePeriod, Resource, ResourceState } from '../../common/lib/types';
+import api from '../../common/utils/api/api';
+import CreateNewOpeningPeriodPage from './CreateNewOpeningPeriodPage';
+
+const testResource: Resource = {
+  id: 1186,
+  name: {
+    fi: 'Test resource name in finnish',
+    sv: 'Test resource name in swedish',
+    en: 'Test resource name in english',
+  },
+  address: {
+    fi: 'Test address in finnish',
+    sv: 'Test address in swedish',
+    en: 'Test address in english',
+  },
+  description: {
+    fi: 'Test description in finnish',
+    sv: 'Test description in swedish',
+    en: 'Test description in english',
+  },
+  extra_data: {
+    citizen_url: 'kansalaisen puolen url',
+    admin_url: 'admin puolen url',
+  },
+};
+
+const testDatePeriod: DatePeriod = {
+  id: 1,
+  created: '2020-11-20',
+  modified: '2020-11-20',
+  is_removed: false,
+  name: { fi: '', sv: '', en: '' },
+  description: { fi: '', sv: '', en: '' },
+  start_date: '',
+  end_date: '2020-11-21',
+  resource_state: ResourceState.OPEN,
+  override: false,
+  resource: 1,
+  time_span_groups: [],
+};
+
+describe(`<CreateNewOpeningPeriodPage />`, () => {
+  async function fillCompulsoryFormFields(container: Element): Promise<void> {
```

Comment:
Can this function locate outside describe block?

Reply:
Probably true, and I actually think it could make the tests more readable, I'll try to change it!

- [ ] Move fillCompulsoryFormFields function outside of describe block